query.md

```diff
@@ -0,0 +1,88 @@
+1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
+
+filter: {name: {\$eq: "Babelgum"}}
+project: {name: 1}
+
+2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
+
+   filter: {number_of_employees: {\$gt: 5000}}
+   sort: -1
+   limit: 20
+
+3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
+   filter: {$and:[{founded_year: {$gte: 2000}},{founded_year: {\$lte: 2005}}]}
+   project: {name: 1}, {founded_year: 1}
+
+4. All the companies that had a Valuation Amount of more than 100.000.000 and have been founded before 2010. Retrieve only the `name` and `ipo` fields.
+   filter: {$and: [{total_money_raised: {$gt: 100000000}},{founded_year: {\$lt: 2010}}]}
+   project: {name: 1}, {ipo: 1}
+
+5. All the companies that have less than 1000 employees and have been founded before 2005. Order them by the number of employees and limit the search to 10 companies.
+
+   filter: {$and: [{number_of_employees: {$lt: 1000}}, {founded_year: {\$lt: 2005}}]}
+   sort:{number_of_employees: -1}
+   limit: 10
+
+6. All the companies that don't include the `partners` field.
+
+   filter: {partners: {\$size: 0}}
+
+7. All the companies that have a null type of value on the `category_code` field.
+
+   filter: {category_code: {\$type: "null"} }
+
+8. All the companies that have at least 100 employees but less than 1000. Retrieve only the `name` and `number of employees` fields.
+
+   filter: {$and: [{number_of_employees: {$gt: 100}},{number_of_employees: {\$lt: 1000}}]}
+   project: {name: 1},{number_of_employees: 1}
+
+9. Order all the companies by their IPO price in a descending order.
+   sort: {"ipo.valuation_amount": -1}
+
+10. Retrieve the 10 companies with more employees, order by the `number of employees`
+
+    sort: {number_of_employees: -1}
+    limit: 10
+
+11. All the companies founded on the second semester of the year. Limit your search to 1000 companies.
+
+    filter: {founded_month:6 }
+    limit: 1000
+
+12. All the companies founded before 2000 that have an acquisition amount of more than 10.000.000
+
+    filter: {$and: [{founded_year: {$lt: 2000}}, {"acquisitions.price_amount": {\$gt: 10000000}}]}
+
+13. All the companies that have been acquired after 2010, order by the acquisition amount, and retrieve only their `name` and `acquisition` field.
+
+    filter: {"acquisition.acquired_year": {\$gt: 2010}}
+    project: {name: 1},{acquisitions: 1}
+    sort: {"acquisition.price_amount": -1}
```

Comment:
you could also decided to sort bu ascending number:  {"acquisition.price_amount":1}